forte/models/da_rl/generator_trainer.py

```diff
@@ -0,0 +1,398 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A wrapper adding data augmentation to a Bert model with arbitrary tasks.
+"""
+
+import random
+import math
+from typing import Tuple, Dict
+import texar.torch as tx
+import torch
+import torch.nn as nn
+from torch.nn import functional as F
+from torch.optim import Optimizer
+
+from forte.models.da_rl.magic_model import MetaModule
+
+
+__all__ = [
+    "MetaAugmentationWrapper"
+]
+
+
+class MetaAugmentationWrapper:
+    # pylint: disable=line-too-long
+    r"""
+    A wrapper adding data augmentation to a Bert model with arbitrary tasks.
+    This is used to perform reinforcement learning for joint data augmentation
+    learning and model training.
```

Comment:
Although the example is not included, some code examples of how to use this class would be very nice.